### PR TITLE
JijieZou add personal VerifyMotto

### PR DESCRIPTION
**Pull request by AsymmetryChou**
With  #4  I finish the homework according to the requirements.
- add personal verifyMotto function with unittest
- add corresponding CI/CD

### CODE REVIEW

**Comment by QG-phy** [comment]
1.  hard to recognize zjj. please sign your name fully spelled.
2. you did not add a new issue and attach it to this PR.
3. there are confilts, please resolve it!

**Comment by QG-phy** [comment]
The title of this PR is  too vague

**Comment by AsymmetryChou** [comment]
I have fixed the conflicts in my PC and git push to origin. As expected, the PR automatically fixs the conflict.

File: dplc/motto/motto.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 def verifyMotto(P: argparse.Namespace):
     name = P.name
     motto = P.motto
-    is_verified = False
+    is_verified = True
```
Comment (QG-phy) [inline dplc/motto/motto.py]: 这里为什么改成默认 True？

File: dplc/motto/motto.py [hunk 2]
```diff
@@ -170,6 +172,28 @@ def verifyMottoQiangqiangGu(public_string: str) -> bool:
...
     is_verified = False
+
     if md5_string == "48f3cb54333cbceda32156f4f2a7dd33":
         is_verified = True
+    return is_verified
+
+def verifyMottoJijieZou(public_string: str) -> bool:
+    '''
+
+    Verify if a public string is Qiangqiang Gu's motto.
```
Comment (QG-phy) [inline dplc/motto/motto.py]: your docstring is not right

File: dplc/motto/motto.py (hunk 1 above)
Comment (AsymmetryChou) [reply dplc/motto/motto.py]: oh！Just for test, a stupid mistake~

**Pull request merged**